.isort.cfg

```diff
@@ -0,0 +1,4 @@
+[settings]
+known_third_party = chai,dateutil,pytz,setuptools,simplejson
+known_first_party = arrow
```

Comment:
you shouldn't need this line I think (?) `isort` should sort the same without it

Reply:
Just tried it with only the `known_third_party` line and it still worked the same, so I cleaned up all the other settings :). Thanks for the feedback.